src/test/java/seedu/tp/commands/PriorityCommandTest.java

```diff
@@ -0,0 +1,73 @@
+package seedu.tp.commands;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import seedu.tp.exceptions.InvalidFlashcardIndexException;
+import seedu.tp.ui.Ui;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.tp.utils.ExampleInputConstants.FULL_FLASHCARD_LIST;
+import static seedu.tp.flashcard.Flashcard.PriorityLevel;
+
+public class PriorityCommandTest {
+    private final ByteArrayOutputStream capturedOut = new ByteArrayOutputStream();
+    private final PrintStream backupStdout = System.out;
+
+    @BeforeEach
+    public void captureStdout() {
+        System.setOut(new PrintStream(capturedOut));
+    }
+
+    @AfterEach
+    public void restoreStdout() {
+        System.setOut(backupStdout);
+    }
+
+    @Test
+    public void priorityCommand_execute_setDefaultPrioritySuccessfully() throws InvalidFlashcardIndexException {
+        StringBuilder expectedEventOutputDefault = new StringBuilder();
+        expectedEventOutputDefault.append("Priority has been updated:" + System.lineSeparator());
+        expectedEventOutputDefault.append("Event 1 | New priority: Not indicated" + System.lineSeparator());
+
+        PriorityCommand priorityCommand = new PriorityCommand(FULL_FLASHCARD_LIST, 0, new Ui(), PriorityLevel.DEFAULT);
```

Comment:
Use copy constructor for `FULL_FLASHCARD_LIST` to avoid affecting other tests

Reply:
I'm not sure why the build is still failing. I stopped using FULL_FLASHCARD_LIST in ReviewedCommand and PriorityCommand, and created a new list instead, so it shouldn't be modifying FULL_FLASHCARD_LIST.

Reply:
Hmmmm that's weird. Could you try running the tests in your local and see what are the expected and actual output?

Reply:
Thanks for all the help, the checks finally passed...